index.html

```diff
@@ -0,0 +1,141 @@
+            </div>
+        </header>
+        <section class="home-page container">
+            <h3>"Hello! Sharing world"</h3>
+            <h1><b>TECHNOLOGICAL EDUCATION, GLOBAL SUMMIT (TEGS) 2022</b></h1>
+            <p class="home-p">The education of individuals across the globe on the uses, 
+                importance and benefits of technology and how to adapt to its fast changing nature
+            </p>
+            <h2><b>2022.12.19(MON) ~ 20(TUES)</b></h2>
+            <p><b>@ National Museum of Nigeria, Technological center Owerri and more</b></p>
+        </section>
+    </div>
+    <section class="main-program" id="main-program">
+        <div>
+            <h2 class="text-light">Main Program</h2>
+            <div class="orange-line"></div>
+        </div>
+        <ul>
+            <li class="list">
+                <i class="material-icons">school</i>
+                <h3>Lecture</h3>
+                <p class="text-light">Listen to speakers from various countries about the message of sharing and opening.
+                </p>
+            </li>
+            <li class="list">
+                <i class="material-icons">museum</i>
+                <h3>TEGS Exihibition</h3>
+                <p class="text-light">Appreciate various creations applying TEGS lisence of technologists, organized from tecnological center Owerri.
+                </p>
+            </li>
+            <li class="list">
+                <i class="material-icons">forum</i>
+                <h3>Forum</h3>
+                <p class="text-light">Try creating your own work using open source lisence rather than just watching.</p>
+            </li>
+            <li class="list">
+                <i class="material-icons">home_repair_service</i>
+                <h3>Workshop</h3>
+                <p class="text-light">Try creating your own work using open source lisence rather than just watching.</p>
+            </li>
+            <li class="list">
+                <i class="material-icons">attractions</i>
+                <h3>TEGS Ignite</h3>
+                <p class="text-light">Get the opportunities to network with TEGS affliates around the world, also after the summit.</p>
+            </li>
+        </ul>
+        <button>Join TEGS Global Summit 2022</button>
+        <article class="see-program text-light" id="see-program"><i class="material-icons">arrow_right_alt</i> click here to .<a href="#speakers" class="text-light"><b><u> SEE THE FEATURED SPEAKERS</u></b></a>
+        </article>
+    </section>
+    <!-- Featured Speakers -->
+    <section class="speakers hidden" id="speakers">
+    </section>
+    <section class="partners hidden" id="partners">
+        <div>
+            <h2 class="text-light">Partner</h2>
+            <div class="orange-line"></div>
+        </div>
+        <ul>
+            <li>Mozilla</li>
+            <li>Google</li>
+            <li>NAVER</li>
+            <li>Addidas</li>
+            <li>Mitchelin</li>
+        </ul>
```

Comment:
- [ OPTIONAL] It would be nice to make the Partner section look identical to that of the template provided on both the mobile and desktop view. There is an `opacity` on the text so they don't look a little dim. I would advise you to add an `opacity` of `0.6` or `0.7`. Also, note the layout of the partner's section on the mobile view

### Your Design (Desktop)

![image](https://user-images.githubusercontent.com/77106962/210636333-b95551fc-4c83-4a55-8907-bc0cdb4e4f8a.png)

### Behance Design (Desktop)

![image](https://user-images.githubusercontent.com/77106962/210636211-4fd3e784-c4de-403b-a494-386c0d7e2de1.png)

### Your Design (Mobile)

![image](https://user-images.githubusercontent.com/77106962/210636638-cc903e80-4a48-41e1-ac23-99ce2ae0c118.png)

### Behance Design (Mobile)

![image](https://user-images.githubusercontent.com/77106962/210636809-a33ed30c-6666-48ea-8246-b5bf8e9720fb.png)

#### It would be nice to implement these changes as it would make your application look better and identical to the Behance template. 👍